editor/editor_properties_array_dict.cpp

```diff
@@ -358,8 +358,13 @@ void EditorPropertyArray::update_property() {
 			prop->connect("object_id_selected", this, "_object_id_selected");
 			if (array.get_type() == Variant::ARRAY) {
 				HBoxContainer *hb = memnew(HBoxContainer);
```

Comment:
We now create this hbox no matter what, do we mind if I make this above the if statement?

Reply:
Well it looks like you can move all of it outside the `if` and remove the `else`, and just keep `if (array.get_type() == Variant::ARRAY && subtype == Variant::NIL)` for the only Array-specific logic.

Reply:
Also made sure the edit button was always before remove button:

![image](https://user-images.githubusercontent.com/1107474/59763231-7ed72480-92dc-11e9-9b5d-094ba96da50b.png)